### PR TITLE
Doc changes - Azure WebApps

### PR DESCRIPTION
**Pull request by ankurch627**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[SUMO-243953](https://sumologic.atlassian.net/browse/SUMO-243953)

### CODE REVIEW

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@ankurch627, this PR adds a new file "azure-webapps.md" when file "webapps.md" already exists for Azure Web Apps. Instead of adding the new file, I suggest you edit the existing file.

**Review by jpipkin1 — DISMISSED** [review]
@ankurch627, I made some minor layout changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4571/commits/ccb34644270dc387d16df16e4f8d1e59a95a4f98)").

The dashboard images don't load in my local build. When I go to the URLs, it says "AccessDenied". Do you need to upload the images to S3 and make them public (like you needed to do for [this PR](https://github.com/SumoLogic/sumologic-documentation/pull/4336#discussion_r1690999983))?

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> @ankurch627, I made some minor layout changes (see commit "[Updates from review](https://github.com/SumoLogic/sumologic-documentation/pull/4571/commits/ccb34644270dc387d16df16e4f8d1e59a95a4f98)").
> 
> The dashboard images don't load in my local build. When I go to the URLs, it says "AccessDenied". Do you need to upload the images to S3 and make them public (like you needed to do for [this PR](https://github.com/SumoLogic/sumologic-documentation/pull/4336#discussion_r1690999983))?

@jpipkin1 I'm seeing the same issue on my local build

File: docs/integrations/microsoft-azure/web-apps.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ For more information, see [Azure Web Apps](https://azure.microsoft.com/en-us/ser
 The Azure Web Apps app supports:
 * **Web Server Logging.** Information about HTTP transactions using the [W3C extended log file format](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx). This is useful when determining overall site metrics such as the number of requests handled or how many requests are from a specific IP address.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:19]: web server logging and application diagnostic logs they are same as http logs , AppLogs and console logs so we can remove them.
lets have separate section for Resource logs and Activity Logs similar to https://help.sumologic.com/docs/integrations/microsoft-azure/azure-storage/#log-and-metric-types 
all the log types coming in diagnostic settings should be under resource logs

File: docs/integrations/microsoft-azure/web-apps.md [hunk 2]
```diff
@@ -52,7 +62,7 @@ In this step, you configure a pipeline for shipping logs from [Azure Monitor](ht
     1. Diagnostic Settings blade will show all your existing settings if any already exist. Click **Edit Setting** if you want to change your existing settings, or click **Add diagnostic setting** to add a new one.
     1. Select the **Stream to an event hub box** checkbox.
     1. Select an Azure subscription.
-    1. **Event bub namespace.** If you have chosen Method 1 (Azure Event Hubs Source) for collecting logs, select the **EventHubNamespace** created manually, or else if you have chosen Method 2 (Collect logs from Azure monitor using Azure functions), then select `SumoAzureLogsNamespace<UniqueSuffix>` namespace created by the ARM template.
+    1. **Event hub namespace.** If you have chosen Method 1 (Azure Event Hubs Source) for collecting logs, select the **EventHubNamespace** created manually, or else if you have chosen Method 2 (Collect logs from Azure monitor using Azure functions), then select `SumoAzureLogsNamespace<UniqueSuffix>` namespace created by the ARM template.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:65]: Remove this section ## Collecting logs for Azure Web Apps, we already have configure logs section later, replace it with a setup section https://help.sumologic.com/docs/integrations/microsoft-azure/azure-storage/#setup

File: docs/integrations/microsoft-azure/web-apps.md [hunk 3]
```diff
@@ -78,6 +88,130 @@ In Step 1, you create an HTTP source. When you configure the, plan your source c
...
+| if (!isBlank(resourceLocation), resourceLocation, location) as location
+| if (!isBlank(service_region), service_region, location) as location
+| if (isBlank(location), "global", location) as location
+| fields location
+```
+
+#### Resource ID Extraction FER
+
+```sql
+Rule Name: AzureResourceIdExtractionFER
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:127]: lets update this FER in other azure docs as well.

File: docs/integrations/microsoft-azure/web-apps.md [hunk 4]
```diff
@@ -78,6 +88,130 @@ In Step 1, you create an HTTP source. When you configure the, plan your source c
...
+### Configure logs collection
+
+#### Diagnostic logs
+
+In this section, you will configure a pipeline for shipping diagnostic logs from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) to an Event Hub.
+
+1. To set up the Azure Event Hubs source in Sumo Logic, refer to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+1. To create the **Diagnostic setting** in the Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the below steps for each Azure WebApps that you want to monitor.
+   1. Choose `Stream to an event hub` as the destination.
+   1. Select `AllMetrics`.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:189]: this is logs section why AllMetrics?

File: docs/integrations/microsoft-azure/web-apps.md (hunk 2 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:65]: Updated replaced with setup instructions

File: docs/integrations/microsoft-azure/web-apps.md (hunk 1 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:19]: Updated.

File: docs/integrations/microsoft-azure/web-apps.md (hunk 3 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:127]: Done, updated in all the other azure docs.

File: docs/integrations/microsoft-azure/web-apps.md (hunk 4 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:189]: Updated with all the available log types.

File: docs/integrations/microsoft-azure/web-apps.md [hunk 5]
```diff
@@ -13,11 +13,24 @@ The Azure Web Apps app allows you to collect Azure web server and application di
...
+    * [AppServiceFileAuditLogs](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/appservicefileauditlogs)
+    * [AppServiceIPSecAuditLogs](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/appserviceipsecauditlogs)
+    * [AppServicePlatformLogs](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/appserviceplatformlogs)
+    * [AppServiceAntivirusScanAuditLogs](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/appserviceantivirusscanauditlogs)
+
+    Requests made by the Blob storage service itself, such as log creation or deletion, aren't logged. For a full list of the logged data, see [Storage logged operations and status messages](https://learn.microsoft.com/en-us/rest/api/storageservices/storage-analytics-logged-operations-and-status-messages).
+
+- **Activity logs**, provides insight into any subscription-level or management group level events that have occurred in the Azure. To learn more, refer to [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema).
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:33]: Need to add metrics also

File: docs/integrations/microsoft-azure/web-apps.md [hunk 6]
```diff
@@ -39,44 +52,141 @@ _sourceCategory=Azure/Web-app
...
+* Activity Logs collection from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) using our [Azure Event Hubs source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/). It is recommended to create a separate source for activity logs. If you are already collecting these logs, you can skip this step.
+* Metrics collection using our [HTTP Logs and Metrics source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/) via Azure Functions deployed using the ARM template.
+
+You must explicitly enable diagnostic settings for each storage service (blob,queue,table and file) and each storage account that you want to monitor. You can forward logs to the same event hub provided they satisfy the limitations and permissions as described [here](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?tabs=portal#destination-limitations).
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:63]: storage service (blob,queue,table and file) and each storage account -> web app

File: docs/integrations/microsoft-azure/web-apps.md [hunk 7]
```diff
@@ -39,44 +52,141 @@ _sourceCategory=Azure/Web-app
...
+
+When you configure the event hubs source or HTTP source, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: `Azure/Storage/Logs`, `Azure/Storage/Metrics`.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:65]: Storage -> WebApps

File: docs/integrations/microsoft-azure/web-apps.md [hunk 8]
```diff
@@ -13,11 +13,24 @@ The Azure Web Apps app allows you to collect Azure web server and application di
...
-* **Application Diagnostics Logs.** Application diagnostics allows you to capture information produced by a web application. ASP.NET applications can use the [System.Diagnostics.Trace](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx) class to log information to the application diagnostics log.
+For Azure Web Apps, you can collect the following logs and metrics:
+
+- **Resource logs**, which provide an insight into operations that were performed within an Azure resource. For a complete schema for resource logs refer to the below documentation:
+    * [Web Server Logging](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx)
+    * [Application Diagnostics Logs](http://msdn.microsoft.com/library/windows/desktop/aa814385.aspx)
+    * [Activity logs](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/activity-log-schema)
+    * [AppServiceEnvironmentPlatformLogs](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/tables/appserviceenvironmentplatformlogs)
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:24]: serviceenvironplatformlogs I think these do not come for web apps?

File: docs/integrations/microsoft-azure/web-apps.md [hunk 9]
```diff
@@ -39,44 +52,141 @@ _sourceCategory=Azure/Web-app
...
+1. [Configure an HTTP Source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-1-configure-an-http-source).
+1. [Configure and deploy the ARM Template](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-2-configure-azure-resources-using-arm-template).
+1. [Export metrics to Event Hub](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/#step-3-export-metrics-for-a-particular-resource-to-event-hub). Perform below steps for each Azure WebApps that you want to monitor.
+   1. Choose `Stream to an event hub` as destination.
+   1. Select `AllMetrics`.
+   1. Use the Event Hub namespace created by the ARM template in Step 2 above. You can create a new Event Hub or use the one created by ARM template. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+
+### Configure logs collection
+
+#### Diagnostic logs
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/web-apps.md:158]: Add screenshot for both similar to how sachin has done https://github.com/SumoLogic/sumologic-documentation/pull/4595/files

Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:158]: Added screenshots and updated text

File: docs/integrations/microsoft-azure/web-apps.md (hunk 8 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:24]: Removed

File: docs/integrations/microsoft-azure/web-apps.md (hunk 7 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:65]: Done

File: docs/integrations/microsoft-azure/web-apps.md (hunk 6 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:63]: Done

File: docs/integrations/microsoft-azure/web-apps.md (hunk 5 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/web-apps.md:33]: Added

Comment (himanshu219) [reply docs/integrations/microsoft-azure/web-apps.md:30]: specific -> platform

**Review by himanshu219 — APPROVED** [review]

**Comment by jpipkin1** [comment]
@kimsauce, the dashboard images still don't load for me (see [comment](https://github.com/SumoLogic/sumologic-documentation/pull/4571#pullrequestreview-2326569409) above). Work with @ankurch627 to ensure the images work before publishing this PR.

**Comment by himanshu219** [comment]
@kimsauce @jpipkin1  this one we can wait till tomorrow @ankurch627 is doing some changes and will reupload the screenshots

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Hi @ankurch627 , since the dashboard images have been delaying the release of another document (September apps recap), I've temporarily commented them out and merged the changes. I'll go ahead and open a new ticket to address these dashboard images separately.

Thank you for your understanding!

**Pull request merged**